### ISSUE TITLE
Tweedie Regression

### ISSUE THREAD

**Issue by Sandy4321** [catboost/catboost#817]
Problem: {unbalanced data , do you pay attention to unbalanced data. Your competitors do https://www.kaggle.com/c/allstate-claims-severity/discussion/25208.  xgboost now supports Tweedie Regression for insurance applications . and https://stackoverflow.com/questions/48390484/lightgbm-with-tweedie  What do you have such an algorithms    }
catboost version: {version}
Operating System: {OS}
CPU: {CPU}
# GPU: {GPU}

**Comment by annaveronika** [comment]
Thank you very much for the suggestion! This is a nice idea for contribution, so I'm adding help wanted label to the issue.

**Comment by annaveronika** [comment]
duplicate of #577
